packages/steiger/migrations/transform-config.js

```diff
@@ -0,0 +1,46 @@
+/** @type {import('jscodeshift').Transform} */
+export default function transformer(file, api) {
+  const j = api.jscodeshift
+  const root = j(file.source)
+
+  // Add "fsd/" prefix to FSD rule names
+  root
+    .find(j.Literal)
+    .filter((path) => ruleNames.includes(path.node.value))
+    .replaceWith((path) => j.stringLiteral(`fsd/${path.node.value}`))
+
```

Comment:
Optional. We could check if the config is already in the new shape, just in case. But it seems like that would be a very rare edge case.

Reply:
If it already is, then it doesn't make sense for the user to run the migration, I think they'll know better. So let's just assume that there is something worth migrating